bob-tools/bin/config_cluster_generator/center.rs

```diff
@@ -0,0 +1,597 @@
+use anyhow::{anyhow, Result as AnyResult};
+use bob::{
+    ClusterConfig, ClusterNodeConfig as ClusterNode, ClusterRackConfig as ClusterRack, DiskPath,
+    ReplicaConfig as Replica, VDiskConfig as VDisk,
+};
+use std::{
+    collections::{HashMap, HashSet},
+    ops::AddAssign,
+};
+
+const REPLICA_IN_FIRST_RACK: usize = 2;
+
+#[derive(Debug, Default)]
+struct Counter {
+    racks_used_count: HashMap<String, usize>,
```

Comment:
IMO it's better to use type aliases here (like `type RackName = String`), it'll be more readable